### ISSUE TITLE
Enabling monkey patch changes the install to "OSS Dev"

### ISSUE THREAD

**Issue by wrightwriter** [iocave/monkey-patch#66]
This means no extensions store and a separate new .vscode folder in appdata (windows), so all your settings.json and stuff is still in the old .vscode folder... This happened after I updated my 7-8 months old VSCode to the current version. Tried reinstalling, doesn't help.

**Comment by Samhhali** [comment]
Same problem here too, extensions also disappear and vs code not working well

**Comment by wrightwriter** [comment]
> Same problem here too, extensions also disappear and vs code not working well

Temporary fix is to open a project with VSCode instead of opening VSCode without a project. For some reason this opens the working patched version!

But don't bother reinstalling, doesn't fix it.

**Comment by adamalbrecht** [comment]
Also running into this issue. It seems to open by default pointing to a separate settings directory at `~/Library/Application Support/Code - OSS Dev` (on Macos).

Running it from the command line using `code --user-data-dir ~/Library/Application\ Support/Code` worked for me.

But then I deleted that new directory and re-installed and now it seems to be working again 🤞

**Comment by frzkn** [comment]
I'm also facing this issue, Ventura 13.0.1
Tried @adamalbrecht 's workaround but it didn't work

**Comment by Samhhali** [comment]
> I'm also facing this issue, Ventura 13.0.1 Tried @adamalbrecht 's workaround but it didn't work

Hi!
Reinstalling vs code using Snap worked for me I'm Ubunto 22.04

**Comment by LeoDog896** [comment]
Yes, I've encountered the same issue. Reinstall vscode and uninstall the extension for now.

**Comment by Hobart2967** [comment]
Happens on a fresh install on MacOS (latest, Ventura 13.4.1)

1. Install vscode using brew
2. Install extension
3. Activate
4. Voilá -> OSS version -.-

Only reinstalling helps, will not use the extension anymore further until fixed.

**Comment by Hobart2967** [comment]
After a first analysis, I found out that the bootstrap amd file changed in the native implementation. Though, when updating my local implementation locally, it would run through (Using the original template, adding the monkey-patch "if" and adjusting callback.apply to callback?.apply (because of a null reference issue)

Though, my VSCode instance waits forever. My assumption is, that the patched main.js file does not fit the latest native version anymore as well.

**Comment by Hobart2967** [comment]
Workaround found!

- Open VS Code without Monkey patch enabled or installed.
- Activate Monkey patch, but do not restart yet.
- Open bootstrap-amd.js of your VSCode installation
  - On Mac, this is: /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
  - Copy the part (approximately around line 50, you have to make up the line in your install on your own)
     On Mac, this was looking like (Will be differing for other OS):
     ```js
     paths : {
       "monkey": "[Your User Dir]/Library/Application Support/Code/User/globalStorage/iocave.monkey-patch/modules",
     }
     ```
- Copy the new file contents from here: https://raw.githubusercontent.com/lehni/monkey-patch/master/data/bootstrap-amd.js
- Replace this block with the recently copied configuration:
  ```js

**Comment by Hobart2967** [comment]
Workaround found!

- Open VS Code without Monkey patch enabled or installed.
- Activate Monkey patch, but do not restart yet.
- Open bootstrap-amd.js of your VSCode installation
  - On Mac, this is: /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
  - Copy the part (approximately around line 50, you have to make up the line in your install on your own)
     On Mac, this was looking like (Will be differing for other OS):
     ```js
     paths : {
       "monkey": "[Your User Dir]/Library/Application Support/Code/User/globalStorage/iocave.monkey-patch/modules",
     }
     ```
- Copy the new file contents from here: https://raw.githubusercontent.com/lehni/monkey-patch/master/data/bootstrap-amd.js
- Replace this block with the recently copied configuration:
  ```js
   paths : {
     "monkey": "[[MONKEY_PATCH_ROOT]]",
   }
  ```
  
 Now restart VSCode and you should see the monkey-patch extension active! 🎉🥳
 
 Thanks to @lehnim for patching to the latest version! Would love to see a pull request from your side!

**Comment by Hobart2967** [comment]
Workaround found!

- Open VS Code without Monkey patch enabled or installed.
- Activate Monkey patch, but do not restart yet.
- Open bootstrap-amd.js of your VSCode installation
  - On Mac, this is: /Applications/Visual Studio Code.app/Contents/Resources/app/out/bootstrap-amd.js
  - Copy the part (approximately around line 50, you have to make up the line in your install on your own)
     On Mac, this was looking like (Will be differing for other OS):
     ```js
     paths : {
       "monkey": "[Your User Dir]/Library/Application Support/Code/User/globalStorage/iocave.monkey-patch/modules",
     }
     ```
- Copy the new file contents from here: https://raw.githubusercontent.com/lehni/monkey-patch/master/data/bootstrap-amd.js
- Replace this block with the recently copied configuration:
  ```js
   paths : {
     "monkey": "[[MONKEY_PATCH_ROOT]]",
   }
  ```
  
 Now restart VSCode and you should see the monkey-patch extension active! 🎉🥳
 
 Thanks to @lehni for patching to the latest version! Would love to see a pull request from your side!

**Comment by lehni** [comment]
@Hobart2967 you're welcome! I've created a PR now.
